src/tests/ftest/util/dmg_utils.py

```diff
@@ -321,6 +321,43 @@ def storage_query_list_pools(self, uuid=None, rank=None, verbose=False):
             ("storage", "query", "list-pools"), uuid=uuid, rank=rank,
             verbose=verbose)
 
+    def storage_identify_vmd(self, uuid=None, verbose=False):
+        """Get the result of the 'dmg storage identify vmd".
+
+        Args:
+            uuid (str): Device UUID to query. Defaults to None.
+            verbose (bool, optional): create verbose output. Defaults to False.
+
+        Returns:
+            dict: JSON formatted dmg command result.
+
+        Raises:
+            CommandFailure: if the dmg storage query command fails.
+
+        """
+        return self._get_json_result(
+            ("storage", "identify", "vmd"), uuid=uuid,
+            verbose=verbose)
+
+    def storage_replace_nvme(self, old_uuid=None, new_uuid=None, no_reint=False):
```

Comment:
Both old_uuid and new_uuid required?

Reply:
yes, it is required. Is something wrong here? I used same template as other storage dmg methods.

Reply:
Done.